direnvrc

```diff
@@ -155,4 +163,13 @@ use_devenv() {
     log_status "using cached devenv shell"
     _nix_import_env "$profile_rc"
   fi
+
+  # Watch extra files configured in devenv.nix
+  # This picks up new files and reloads the shell on initial run
+  # TODO: can compare files in files_to_watch with the files we have now
+  if [[ -f "$extra_files_watchlist" ]]; then
+    for file in $(cat "$extra_files_watchlist"); do
+      watch_file "$file"
```

Comment:
Again I would call files_to_watch here and just centralise deduplication in there.